src/components/Pricing.tsx

```diff
@@ -36,7 +37,8 @@ export const Pricing = () => {
             </div>
             <div className="mt-20 flex flex-col flex-wrap items-center justify-center gap-4 lg:flex-row">
               <HobbyPlan />
-              <TeamPlan />
+              {/* <TeamPlan /> */}
```

Comment:
Do we want to just delete the team plan?

Reply:
Deleted